soco/events.py

```diff
@@ -576,6 +575,13 @@ def time_left(self):
             time_left = self.timeout-(time.time()-self._timestamp)
             return time_left if time_left > 0 else 0
 
+# if you want to use a different port than 1400, set
+# soco.events.event_listener_port accordingly after importing but before
+# subscribing to an event
+
+# pylint: disable=C0103
+event_listener_port = 1400
```

Comment:
Well, the `start` method is not called from the outside, but from the inside when a user subscribes to an event. How would you want to pass the parameter for the port from the outside then? Should I make the port a `subscribe` parameter? I don't think that's better.

I agree moving the port config to soco.config is better though.